src/library/scala/StringContext.scala

```diff
@@ -127,7 +127,7 @@ case class StringContext(parts: String*) {
      *  Here, we use the `TimeSplitter` regex within the `s` matcher, further splitting the
      *  matched string "10.50" into its constituent parts
      */
-    def unapplySeq(s: String): Option[Seq[String]] = glob(parts, s)
+    def unapplySeq(s: String): Option[Seq[String]] = glob(parts.map(processEscapes), s)
```

Comment:
`processEscapes` can throw a `InvalidEscapeException`, should the exception be handled?

```
scala.StringContext$InvalidUnicodeEscapeException: invalid unicode escape at index 7 of Hello,\u
  at scala.StringContext$.loop$1(StringContext.scala:341)
```

Reply:
👍  I was just confirming.